core/merge.js

```diff
@@ -158,6 +158,18 @@ class Merge {
     }
     metadata.assignMaxDate(doc, file)
     if (file) {
+      if (doc.ino && doc.ino !== file.ino) {
+        await this.resolveConflictAsync(side, doc, file)
+        if (side === 'remote') {
+          // we just renamed the remote file as a conflict
```

Comment:
I think it is dead code: if `side === 'remote'`, the `doc` was built by the Remote Watcher, and it should have no `ino`. Right?

Reply:
You're probably right.